### PR TITLE
Fix default units

### PR DESCRIPTION
**Pull request by selvach**
# Description

- Fixes Swedish defaults to metric units on the AboutYouScreen. 
- Changes Weight for the UK to be in kgs. 

## On what platform have you tested the change?

- [ ] Android - Emulator
- [x] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

### CODE REVIEW

File: src/core/user/UserService.ts [hunk 1]
```diff
@@ -414,3 +414,39 @@ export default class UserService extends ApiClientBase {
...
+  switch (UserService.userCountry) {
+    case 'US':
+    case 'GB':
+      return 'ft';
+    case 'SE':
+      return 'cm';
+    default:
+      return 'cm';
+  }
+};
```
Comment (MikeD-ZOE) [inline src/core/user/UserService.ts:426]: I'd prefer these to be configuration values from `src/core/Config.ts`. Perhaps with the default set to `cm`, and localised US and GB set to `ft`.

File: src/core/user/UserService.ts [hunk 2]
```diff
@@ -414,3 +414,39 @@ export default class UserService extends ApiClientBase {
...
+export const defaultWeightUnit = () => {
+  switch (UserService.userCountry) {
+    case 'US':
+      return 'lbs';
+    case 'GB':
+    case 'SE':
+      return 'kg';
+    default:
+      return 'kg';
+  }
```
Comment (MikeD-ZOE) [inline src/core/user/UserService.ts:434]: Same here. default to `kg`

File: src/core/user/UserService.ts [hunk 3]
```diff
@@ -414,3 +414,39 @@ export default class UserService extends ApiClientBase {
...
+export const defaultTemperatureUnit = () => {
+  switch (UserService.userCountry) {
+    case 'US':
+      return 'F';
+    case 'GB':
+    case 'SE':
+      return 'C';
+    default:
+      return 'C';
+  }
```
Comment (MikeD-ZOE) [inline src/core/user/UserService.ts:443]: Same here. default `C`.

File: src/features/patient/AboutYouScreen.tsx [hunk 4]
```diff
@@ -313,6 +311,10 @@ export default class AboutYouScreen extends Component<AboutYouProps, State> {
       });
     };
 
+    const initValues = cloneDeep(initialFormValues);
+    initValues.heightUnit = defaultHeightUnit();
+    initValues.weightUnit = defaultWeightUnit();
```
Comment (MikeD-ZOE) [inline src/features/patient/AboutYouScreen.tsx:316]:
```
const initValues = cloneDeep({
    ...initialFormValues,
    heightUnit: defaultHeightUnit(),
    weightUnit: defaultWeightUnit()
});
````

File: src/core/user/UserService.ts [hunk 5]
```diff
@@ -420,8 +420,6 @@ export const defaultHeightUnit = () => {
     case 'US':
     case 'GB':
       return 'ft';
-    case 'SE':
-      return 'cm';
     default:
       return 'cm';
   }
```
Comment (MikeD-ZOE) [inline src/core/user/UserService.ts:425]: I'd prefer these to be configuration values from `src/core/Config.ts`. Perhaps with the default set to cm, and localised US and GB set to ft.

Comment (selvach) [reply src/core/user/UserService.ts:425]: In the shower this morning, I thought exactly the same thing :)

Comment (MikeD-ZOE) [reply src/core/user/UserService.ts]: The tricky part is dealing with pounds. UK is stones+pounds, US is just pounds. Hopefully just defaulting Sweden to kg is enough to unblock us going live.

**Comment by selvach** [comment]
Hmpf... this doesn't actually work. setting state in componentDidMount() triggers a re-render, but Formik remembers the old values.

**Comment by selvach** [comment]
Urgh... there's some weird behaviour now, because getConfig is async and the form keeps re-rendering. I'm away until 5pm looking after the kids if someone can pick this up.

**Comment by selvach** [comment]
@MikeD-ZOE - Just saw your PR #187 that will definitively help with this!

File: src/features/assessment/DescribeSymptomsScreen.tsx [hunk 6]
```diff
@@ -80,8 +80,12 @@ export default class DescribeSymptomsScreen extends Component<SymptomProps, Stat
     super(props);
     this.state = initialState;
     this.handleUpdateSymptoms = this.handleUpdateSymptoms.bind(this);
+  }
 
-    initialFormValues.temperatureUnit = isUSCountry() ? 'F' : 'C';
+  async componentDidMount() {
+    const userService = new UserService();
+    const features = await userService.getConfig();
```
Comment (JoshElgar) [inline src/features/assessment/DescribeSymptomsScreen.tsx:87]: It's no longer a promise no?

Comment (MikeD-ZOE) [reply src/features/assessment/DescribeSymptomsScreen.tsx]: no. fixed

**Pull request merged**